codecept.conf.js

```diff
@@ -5,22 +5,24 @@ const baseUrl = `https://${testEnv}`;
 
 exports.config = {
   tests: "tests/**/*.js",
-  timeout: 10000,
+  timeout: 30000,
   output: process.cwd() + '/functional-output',
   helpers: {
     Puppeteer: {
       url: baseUrl,
       show: false,
-      restart: false,
+      restart: true,
```

Comment:
I think once this is true (which is the default btw), `keepCookies` and `keepBrowserState` below don't take effect
https://codecept.io/helpers/Puppeteer/

Reply:
Agreed. If they can be removed then we should do so.